enterprise-suite/es-monitor-api/prometheus.yml

```diff
@@ -145,6 +145,14 @@ scrape_configs:
 
     # workload labels:
     metric_relabel_configs:
+      - source_labels: [__meta_kubernetes_pod_label_pod_template_hash, __meta_kubernetes_pod_name]
+        action: replace
+        regex: '[^;]+;(.*)-[^-]+-[^-]+'
+        target_label: workload
+      - source_labels: [__meta_kubernetes_pod_label_statefulset_kubernetes_io_pod_name]
+        action: replace
+        regex: (.*)-[0-9]+
+        target_label: workload
```

Comment:
Does this work with kube-state-metrics? This entire section actually is meant for kube-state-metrics scraping in the local helm install. We discussed this before and decided we need to make a separate job_name for service scraping, and make this tied to kube-state-metrics. https://github.com/lightbend/es-backend/issues/304 is the issue.

Reply:
this is frankly voodoo to me.  @dbrinegar helped me make these changes so that I could enable metric scraping from localhost.  I didn't need kube-state-metrics for that.  If there is a better way to do it, I'm all for that, but I don't know enough about prom config to do that.